### PR TITLE
Rewrite toolbar

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/gui/CMakeLists.txt [hunk 1]
```diff
@@ -126,6 +126,8 @@ ecm_add_qml_module(owncloudGui
         VERSION 1.0
         NAMESPACE OCC
         QML_FILES
+            qml/AccountBar.qml
```
Comment (TheOneRing) [inline src/gui/CMakeLists.txt:129]: Should I start a new subfolder?

File: src/gui/settingsdialog.cpp [hunk 2]
```diff
@@ -74,207 +57,58 @@ auto minimumSizeHint(const QWidget *w)
...
-    connect(_addAccountAction, &QAction::triggered, this, [] {
-        // don't directly connect here, ocApp might not be defined yet
-        ocApp()->gui()->runNewAccountWizard();
-    });
-    _ui->toolBar->addAction(_addAccountAction);
+    // People perceive this as a Window, so also make Ctrl+W work
+    addAction(tr("Hide"), Qt::CTRL | Qt::Key_W, this, &SettingsDialog::hide);
+
+    _ui->quickWidget->rootContext()->setContextProperty(QStringLiteral("settingsDialog"), this);
+    // TODO: fix sizing
```
Comment (TheOneRing) [inline src/gui/settingsdialog.cpp:80]: Any better idea?

File: src/gui/CMakeLists.txt (hunk 1 above)
Comment (erikjv) [reply src/gui/CMakeLists.txt:129]: No, don't think that's necessary yet.

File: src/gui/settingsdialog.cpp (hunk 2 above)
Comment (erikjv) [reply src/gui/settingsdialog.cpp:80]: No, not yet

**Pull request merged**